tools/ports/sdl2.py

```diff
@@ -90,7 +90,7 @@ def clear(ports, settings, shared):
 
 
 def process_dependencies(settings):
-  settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$autoResumeAudioContext']
+  settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$autoResumeAudioContext', '$dynCall']
```

Comment:
I think it would be safer to always do this for dynCall in all (non-minimal runtime) builds - that is, to do this in emcc.py. That would make this change less noticeable to users.

Separately, removing it from `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE` later might make sense, but it would be good at that later time to document that in the changelog + make sure it gives a clear error to users if they need to add it manually from then onwards.